File: DEV/Backend/restservice/src/main/java/de/dlrg/Restservice/Benutzer.java
package de.dlrg.Restservice;

//import org.springframework.data.annotation.Id;

import javax.persistence.GeneratedValue;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

import javax.persistence.Table;

    @Entity
    @Table(name = "Benutzer")
    public class Benutzer {
        @Id
        @GeneratedValue(strategy = GenerationType.AUTO)
        private Long idrfid;
        private String name;
        private String vorname;
        private double guthaben;
        private int isAdmin;    // 0-> ist kein Admin --  1-> ist Admin
        private Long chipid;

        public Benutzer(Long idrfid, String name, String vorname, double guthaben, int isAdmin, Long chipid) {
            this.idrfid = idrfid;
            this.name = name;
            this.vorname = vorname;
            this.guthaben = guthaben;
            this.isAdmin = isAdmin;
            this.chipid = chipid;
        }



        public Long getIdrfid() {
            return idrfid;
        }

        public void setIdrfid(Long idrfid) {
            this.idrfid = idrfid;
        }

        public String getName() {
            return name;
        }

        public void setName(String name) {
            this.name = name;
        }

        public String getVorname() {
            return vorname;
        }

        public void setVorname(String vorname) {
            this.vorname = vorname;
        }

        public double getGuthaben() {
            return guthaben;
        }

        public void setGuthaben(double guthaben) {
            this.guthaben = guthaben;
        }

        public int getIsAdmin() {
            return isAdmin;
        }

        public void setIsAdmin(int isAdmin) {
            this.isAdmin = isAdmin;
        }

        public Long getChipid() {
            return chipid;
        }

        public void setChipid(Long chipid) {
            this.chipid = chipid;
        }


    }

File: DEV/Client/DB-Zugriff/benutzer.py
class Benutzer(object):
    def __init__(self, idrfid, name, vorname, guthaben, isAdmin, chipid):
        self.Idrfid = idrfid
        self.Name = name
        self.Vorname = vorname
        self.Guthaben = guthaben
        self.IsAdmin = isAdmin
        self.CHIPID = chipid


def get_idrfid(self):
    return self.Idrfid


def get_name(self):
    return self.Name


def get_vorname(self):
    return self.Vorname


def get_Guthaben(self):
    return self.Guthaben


def get_ChipId(self):
    return self.CHIPID
